homeassistant/components/scrape/sensor.py

```diff
@@ -39,38 +40,36 @@
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
+from .const import CONF_INDEX, CONF_SELECT, DEFAULT_NAME
 from .coordinator import ScrapeCoordinator
 
 _LOGGER = logging.getLogger(__name__)
 
 SCAN_INTERVAL = timedelta(minutes=10)
 
-CONF_ATTR = "attribute"
-CONF_SELECT = "select"
-CONF_INDEX = "index"
-
-DEFAULT_NAME = "Web scrape"
-DEFAULT_VERIFY_SSL = True
```

Comment:
`CONF_ATTR` replaced by `CONF_ATTRIBUTE` from `homeassistant.const`
`DEFAULT_VERIFY_SSL` now imported from `homeassistant.components.rest`
The others are now imported from `homeassistant.components.scrape.const`